tests/analysis_tests/tests/src/expected/Completion.res.txt

```diff
@@ -728,26 +686,8 @@ Path ForAuto.
   }]
 
 Complete src/Completion.res 47:21
-posCursor:[47:21] posNoWhite:[47:20] Found expr:[47:3->47:21]
-posCursor:[47:21] posNoWhite:[47:20] Found expr:[47:12->47:21]
-Pexp_ident Js.Dict.u:[47:12->47:21]
-Completable: Cpath Value[Js, Dict, u]
-Package opens Pervasives.JsxModules.place holder
-ContextPath Value[Js, Dict, u]
-Path Js.Dict.u
-[{
-    "label": "unsafeGet",
-    "kind": 12,
-    "tags": [],
-    "detail": "(t<'a>, key) => 'a",
-    "documentation": {"kind": "markdown", "value": "\n`Js.Dict.unsafeGet(key)` returns the value if the key exists, otherwise an `undefined` value is returned. Use this only when you are sure the key exists (i.e. when having used the `keys()` function to check that the key is valid).\n\n## Examples\n\n```rescript\nJs.Dict.unsafeGet(ages, \"Fred\") == 49\nJs.Dict.unsafeGet(ages, \"Paul\") // returns undefined\n```\n"}
-  }, {
-    "label": "unsafeDeleteKey",
-    "kind": 12,
-    "tags": [],
-    "detail": "(t<string>, string) => unit",
-    "documentation": {"kind": "markdown", "value": " Experimental internal function "}
-  }]
+XXX Not found!
+[]
```

Comment:
This change is fine for now. It does highlight a problem, but it's something we'll fix separately.